jobs/gorouter/spec

```diff
@@ -523,6 +523,18 @@ properties:
       Requests from router to backend endpoints that are longer than this duration will be canceled and logged as
       `backend-request-timeout` errors. If set to 0 this timeout is disabled.
     default: 900
+  http_1_request_timeout_in_seconds:
+    description: |
+      This configures an entire request timeout.
+      HTTP/1 requests from router to backend endpoints that are longer than this duration will be canceled and logged as
+      `backend-request-timeout` errors. If set to 0 this timeout is disabled.
+    default: 900
+  http_2_request_timeout_in_seconds:
+    description: |
+      This configures an entire request timeout.
+      HTTP/2 requests from router to backend endpoints that are longer than this duration will be canceled and logged as
+      `backend-request-timeout` errors. If set to 0 this timeout is disabled.
```

Comment:
These should say something like "if set to -1, this timeout is disabled. If set to 0, it inherits the value from `request_timeout_in_seconds`"

Reply:
Couldn't we use set / unset to decide whether to use `request_timeout_in_seconds` or `http_N_request_timeout_in_seconds`? Then we can avoid the additional magic value.